recipes/motus/meta.yaml

```diff
@@ -0,0 +1,25 @@
+package:
+  name: motus
+  version: "1.0"
+
+source:
+  fn: mOTUs.Linux64bits.tar.gz
+  url: http://www.bork.embl.de/software/mOTU/share/mOTUs.Linux64bits.tar.gz
```

Comment:
This indicates something is compiled, correct? Can this be compiled in the build.sh script?